_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,475 @@
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has an icon that makes it distinguishable as a link.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a.test {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+</style>
+<p>
+	Read about WAI on the
+	<a class="test" href="http://w3.org/WAI">WAI webpage <img src="/test-assets/be4d0c/icon.png" alt="" /></a>.
+</p>
+```
+
+#### Passed Example 10
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has text that makes it distinguishable as a link.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a.test {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+</style>
+<p>To read about WAI, <a class="test" href="http://w3.org/WAI">follow this link to the WAI webpage</a>.</p>
+```
+
+#### Passed Example 11
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has an icon that appears when the link is hovered. This icon indicates the element is a link.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+	.hide {
+		display: none;
+	}
+	a:hover img.hide {
+		display: inline;
+	}
+</style>
+<p>
+	Read about WAI on the
+	<a href="http://w3.org/WAI">WAI webpage <img class="hide" src="/test-assets/be4d0c/icon.png" alt="" /></a>.
+</p>
+```
+
+#### Passed Example 12
+
+This link had a descendant element that is distinguishable by color (hue) from the other text in the same line and has a distinguishable bottom border when it is hovered or focused.
+
+```html
+<style>
+	a {
+		text-decoration: none;
+	}
+
+	a:hover span,
+	a:focus span {
+		border-bottom: 2px solid currentColor;
+	}
+</style>
+
+<p>
+	Read about WAI on the <a href="http://w3.org/WAI"><span>WAI webpage</span></a
+	>.
+</p>
+```
+
+#### Passed Example 13
+
+This link is distinguishable by color (hue) from the other text in the same line in both link history states. The link has distinguishing color and background color. The color and background color are different when in the visited state, but these are also distinguishing colors.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a:link {
+		color: white;
+		background-color: blue;
+		text-decoration: none;
+	}
+	a:visited {
+		color: blue;
+		background-color: lightgray;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has no other visual cues of being recognized as a link with the underline removed. Even though the link has a different color (blue) from the other text (black) the low difference in contrast between them means they are not distinguishing colors.
```

Comment:
```suggestion
This link, that is distinguishable by color (hue) from the other text in the same line, has no other visual cues of being recognized as a link because the underline has been removed. Even though the link has a different color (blue) from the other text (black) the low difference in contrast between them means they are not distinguishing colors.
```